scripts/eslint-rules/warning-args.js

```diff
@@ -44,17 +44,18 @@ module.exports = function(context) {
       // This could be a little smarter by checking context.getScope() to see
       // how warning/invariant was defined.
       const isWarning =
-        node.callee.type === 'Identifier' &&
-        (node.callee.name === 'warning' ||
-          node.callee.name === 'warningWithoutStack' ||
-          node.callee.name === 'lowPriorityWarning' ||
-          node.callee.name === 'lowPriorityWarningWithoutStack');
+        node.callee.type === 'MemberExpression' &&
+        node.callee.object.type === 'Identifier' &&
```

Comment:
Should probably also check:
```js
node.callee.object.name === "console" &&
```
To avoid potential future collision with e.g.
```js
React.warn(...)
```